CMakeLists.txt

```diff
@@ -203,9 +206,10 @@ if (GALOIS_ENABLE_DIST)
   add_subdirectory(libcusp)
   add_subdirectory(libgluon)
 endif()
+string(COMPARE NOTEQUAL "${GALOIS_CUDA_CAPABILITY}" "" GALOIS_ENABLE_GPU)
 if (GALOIS_ENABLE_GPU)
   enable_language(CUDA)
-  string(REPLACE "." "" GENCODES ${CUDA_CAPABILITY})
+  string(REPLACE "." "" GENCODES ${GALOIS_CUDA_CAPABILITY})
   string(REPLACE "," ";" GENCODES ${GENCODES})
```

Comment:
I wasn't around when `${CUDA_CAPABILITY}` was introduced but was there a reason that we used a comma-separated `,` list rather than semi-colon `;`? The latter is more commonly used to separate list items in CMake and we probably could get rid of the `string(REPLACE "," ";" ...)` below.

Reply:
I'm fine with either. I had used comma-separated because I thought it was more commonly used. However, as we're using semi-colon list for other cmake options, I'll change this option to semi-colon as well to be consistent.